packages/calypso-products/src/types.ts

```diff
@@ -26,8 +26,8 @@ import type { ReactElement } from 'react';
 export type Feature = string;
 
 // WPCom
-export type WPComProductSlug = ( typeof WPCOM_PRODUCTS )[ number ];
-export type WPComPlanSlug = ( typeof WPCOM_PLANS )[ number ];
```

Comment:
IIRC this change was recently added (https://github.com/Automattic/wp-calypso/pull/74775) to conform to an updated version of prettier. We probably shouldn't undo it.

Reply:
Thanks, I'm not sure how it slipped in. Will undo it.